app/src/generate.py

```diff
@@ -10,7 +10,12 @@
 
 PROMPT = """Provide answers in plain language written at the average American reading level.
 Use bullet points. Keep your answers brief, max of 5 sentences.
-Keep your answers as similar to your knowledge text as you can"""
+Keep your answers as similar to your knowledge text as you can
+
+Do not quote the context directly, but instead cite when you are drawing from the context using the provided example citation numbers in parenthesis, e.g., (chunk-0), etc.
+Do not add bold formatting to the citations -- write them exactly as shown e.g., (chunk-0).
+Do not add additional text to the citations like (Citation: chunk-0).
```

Comment:
Try using [in-context learning (ICL)](https://www.lakera.ai/blog/what-is-in-context-learning) by giving examples. I think that will be more reliable than providing a bunch of "Do not ..." instructions, which we may have to add to as the LLM is updated.